### DISCUSSION TITLE
NootRX support

### DISCUSSION THREAD

**Discussion by ivan-ef** [osy86/HaC-Mini#1102] [Q&A]
Is it possible to add NootRX driver for Navi 23 GPU support?
https://github.com/ChefKissInc/NootRX

**Comment by goodbest** [comment]
Our GPU is Polaris 22, instead of Navi 22.

Why to add this?

**Reply by xiaooloong** [reply]
或许是想在雷电上接显卡？

**Reply by ivan-ef** [reply]
> Our GPU is Polaris 22, instead of Navi 22.
> 
> 
> 
> Why to add this?
> 
> 
To use Thunderbolt EGPU

**Reply by ivan-ef** [reply]
> 在

Yes. Thunderbolt EGPU
